ffac-eol-ssid/luasrc/lib/gluon/upgrade/500-eol-ssid-config

```diff
@@ -0,0 +1,12 @@
+#!/usr/bin/lua
+
+local uci = require('simple-uci').cursor()
+
+-- migrate older versions
+if uci:get('eol-wifi', 'ssid') ~= nil then
+    local deprecated_ssid_enabled = not uci:get_bool('eol-wifi', 'ssid', 'disabled')
+    uci:set('eol-ssid', 'settings', 'enabled', deprecated_ssid_enabled)
+    uci:save('eol-ssid')
+    uci:delete('eol-wifi', 'ssid')
+    uci:save('eol-wifi')
```

Comment:
Remove the file then altogether? Also you can combine this file with the other existing one by putting the content of this file in the beginning.

Reply:
Do you mean to just delete the file in /etc/config? I thought that might cause harm. 


I split them in 500 and 950 as they serve different purposes:
- the first just migrates the configs (more coming soon)
- the second does the actual job of changing the SSID (as late as possible)

Reply:
Functionally it makes no difference.

> I thought that might cause harm.

Where is this assumption coming from? In It's current state when you do not remove the file, you don't break the UCI cursor (which you'd do if you were to remove the old config-file).

Reply:
> Where is this assumption coming from?

Lack of understanding. I just did not want to break something. I replaced the two lines with `os.remove('/etc/config/eol-wifi')`.

Reply:
and merged both files